components/hab/src/license.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2018 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
Don't need the copyright boilerplate anymore: https://github.com/habitat-sh/habitat/issues/6286

We're not savages 😉

Reply:
Ah yes.  I wrote this before that decision came down from legal and forgot to remove it.  Will do.